eslint.config.mjs

```diff
@@ -0,0 +1,82 @@
+import globals from 'globals'
+import eslint from '@eslint/js'
+
+import tseslint from 'typescript-eslint'
+import reactPlugin from 'eslint-plugin-react'
+import reactHooksPlugin from 'eslint-plugin-react-hooks'
+import jsxA11yPlugin from 'eslint-plugin-jsx-a11y'
+import testingLibraryPlugin from 'eslint-plugin-testing-library'
+import jestDomPlugin from 'eslint-plugin-jest-dom'
+import importPlugin from 'eslint-plugin-import'
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  ...tseslint.configs.stylistic,
+  reactPlugin.configs.flat.recommended,
+  reactHooksPlugin.configs['recommended-latest'],
+  testingLibraryPlugin.configs['flat/react'],
+  jestDomPlugin.configs['flat/recommended'],
+  importPlugin.flatConfigs.recommended,
+  jsxA11yPlugin.flatConfigs.recommended,
+
+  {
+    settings: {
+      'import/resolver': {
+        node: {
+          extensions: ['.js', '.jsx'],
+        },
+      },
+      'import/extensions': ['.js', '.jsx'],
```

Comment:
Do you need to include .ts and .tsx files here too?

Reply:
good catch! TS is officially out of scope, but its an easy change, why not do it?